src/main/java/ch/njol/skript/bukkitutil/ItemUtils.java

```diff
@@ -142,4 +143,18 @@ public static boolean itemStacksEqual(final @Nullable ItemStack is1, final @Null
 			&& is1.getItemMeta().equals(is2.getItemMeta());
 	}
 	
+	/**
+	 * Check if a material is an item that can be held
+	 *
+	 * @param material Material to check if item
+	 * @return True if material is an item
+	 */
+	public static boolean isItem(Material material) {
```

Comment:
Maybe name it with hasItemForm or something like that. Materials are *not* items themselves.